search/src/Components/upload/upload_stepper.js

```diff
@@ -209,8 +252,14 @@ class UploadStepper extends React.Component {
         })
     }
 
-    handleAgContextExempt(){
-        return this.state.activeStep === 1 && this.props.upload_type === 'coco'
+    handleExempt(){
+        if (this.state.activeStep === 1) {
```

Comment:
(Hmm... it would be good if we could use names for the active step, rather than numbers.)